src/main/java/graphql/util/TraverserState.java

```diff
@@ -140,6 +140,6 @@ public void addVisited(T visited) {
                                                       NodeLocation nodeLocation,
                                                       boolean isRootContext) {
         assertNotNull(vars);
-        return new DefaultTraverserContext<>(curNode, parent, visited, vars, sharedContextData, nodeLocation, isRootContext);
+        return new DefaultTraverserContext<>(curNode, parent, visited, vars, sharedContextData, nodeLocation, isRootContext, false);
     }
```

Comment:
is this meant to be always false??

Reply:
`TraverserState` is only used by non parallel traversing ... so it is always false